src/Arabic.php

```diff
@@ -3872,11 +3872,15 @@ public function volc($olc, $codeLength = 10)
      * @param string  $plural3  Plural form 3 (e.g., عناصر). If NULL [default] retrive from internal JSON dataset.
      * @param string  $plural4  Plural form 4 (e.g., عنصرا). If NULL [default] retrive from internal JSON dataset.
      *
-     * @return string Proper plural form of the given singular form
+     * @return string|\Exception Proper plural form of the given singular form, throw exiption if the $singular is not pre defiend and $plural is not passed
```

Comment:
`@return` tag in the phpdoc should stay as it is, and exceptions should be documented using the  `@throws` tag instead. For more information, please check this reference:
https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/throws.html